gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/InstrumentatedGoogleHadoopFileSystem.java

```diff
@@ -271,6 +281,10 @@ public String getScheme() {
     return null;
   }
 
+  @Override
+  public void close() throws IOException {
+    super.close();
+  }
```

Comment:
Please remove this, it is redundant with the updated changes

Reply:
Updated